From 6506ff046d854e6dbc616346ab2f4954f39f8d30 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Tue, 20 Dec 2022 08:09:13 +0200
Subject: [PATCH 51/63] arm64: dts: Add S32CC compatible to GPIO nodes

This is needed to ease the job of the bootloaders in case
there are device tree fixups that could be applied on S32CC
platforms.

Issue: ALB-9425
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32g.dtsi   | 2 +-
 arch/arm64/boot/dts/freescale/s32r45.dtsi | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/s32g.dtsi b/arch/arm64/boot/dts/freescale/s32g.dtsi
index a23e39f07dc9..c117a69491fd 100644
--- a/arch/arm64/boot/dts/freescale/s32g.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32g.dtsi
@@ -126,7 +126,7 @@ pinctrl: siul2-pinctrl@4009c240 {
 			};
 
 			gpio: siul2-gpio@4009d700 {
-				compatible = "nxp,s32g-siul2-gpio";
+				compatible = "nxp,s32g-siul2-gpio", "nxp,s32cc-siul2-gpio";
 
 				reg = <OPADS_SIUL2_0        0    0x0 0x10>,
 				      <OPADS_SIUL2_1        0    0x0 0x18>,
diff --git a/arch/arm64/boot/dts/freescale/s32r45.dtsi b/arch/arm64/boot/dts/freescale/s32r45.dtsi
index 0a5793e3b92e..40702cf919e5 100644
--- a/arch/arm64/boot/dts/freescale/s32r45.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32r45.dtsi
@@ -83,7 +83,7 @@ pinctrl: siul2-pinctrl@4009c240 {
 			};
 
 			gpio: siul2-gpio@4009d700 {
-				compatible = "nxp,s32r-siul2-gpio";
+				compatible = "nxp,s32r-siul2-gpio", "nxp,s32cc-siul2-gpio";
 
 				reg = <OPADS_SIUL2_0       0 0x0 0x10>,
 				      <OPADS_SIUL2_1       0 0x0 0x14>,
-- 
2.25.1

